azure-pipelines.yml

```diff
@@ -0,0 +1,72 @@
+jobs:
```

Comment:
I take it Azure Pipelines was dramatically better?

Reply:
In my opinion: definitely. Particularly for .NET Core and getting it to run across many platforms.